wandb/sdk/wandb_run.py

```diff
@@ -3186,7 +3233,8 @@ def link_model(
                     - `/local/directory/file.txt`
                     - `s3://bucket/path`
             registered_model_name: (str) - the name of the registered model that the model is to be linked to. The entity will be derived from the run
-            model_name: (str) - the name of the model artifact that files in 'path' will be logged to.
+            name: (str) - the name of the model artifact that files in 'path' will be logged to. This will default to the basename of the path prepended with the current
```

Comment:
Is this argument not optional? I thought we default to the basename of the path prepended with the current run id if the user doesn't specify a name?